specs/protocols/tickers.yaml

```diff
@@ -10,11 +10,14 @@ speech_acts:
   get_all_tickers:
     ledger_id: pt:optional[pt:str]
     exchange_id: pt:optional[pt:str]
-    params: pt:optional[pt:dict[pt:str, pt:bytes]]
+    params: pt:optional[pt:bytes]
```

Comment:
@gauravlochab 
MIght have some small changes here.

I figured out how to get much more accurate prices for fetching individual tickers.

Additionally, - ensured that all the logs are much much cleaner from the connection. 

I dont believe there will be any issues with this being applied to baby-degen etc.